shared/test/capture_queries.rb

```diff
@@ -0,0 +1,35 @@
+require 'stringio'
```

Comment:
This module is a Sequel-variant of the ActiveRecord/Rails-oriented [`CaptureQueries` module](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/test/testing/capture_queries.rb) in Dashboard added in #18403.